src/docs/cookbook/design/tabs.md

```diff
@@ -65,18 +74,20 @@ DefaultTabController(
 );
 ```
 
-By default, the `TabBar` looks up the Widget tree for the nearest
-`DefaultTabController`. If you're manually creating a `TabController`, you'll
-need to pass it to the `TabBar`.
+By default, the `TabBar` looks up the widget tree for the nearest
+`DefaultTabController`. If you're manually creating a `TabController`,
+pass it to the `TabBar`.
 
 ## 3. Create content for each tab
 
-Now that we have tabs, we'll want to display content when a tab is selected.
-For this purpose, we'll employ the
-[`TabBarView`]({{site.api}}/flutter/material/TabBarView-class.html) Widget.
+Now that you have tabs, display content when a tab is selected.
```

Comment:
I'm not crazy about the command to display content. Maybe:

When a tab is selected, it needs to display content.